javascript/ql/test/library-tests/frameworks/Immutable/immutable.js

```diff
@@ -55,4 +55,7 @@ Set.of(source()).filter(x => true).toList().forEach(x => sink(x)); // NOT OK
 
 Set([source()]).filter(x => true).toList().forEach(x => sink(x)); // NOT OK
 
-OrderedSet([source()]).filter(x => true).toList().forEach(x => sink(x)); // NOT OK
\ No newline at end of file
+OrderedSet([source()]).filter(x => true).toList().forEach(x => sink(x)); // NOT OK
+
+x.d; // ensure 'd' property exists
+x.f; // ensure 'f' property exists
```

Comment:
Should we add all string-constants as `PropertyName` instead?

Reply:
I think I'd rather just port the model to flow summaries (in a different PR)